src/components/Inventory/InventoryCellRenderer.tsx

```diff
@@ -17,11 +18,14 @@ export default function InventoryCellRenderer(props: RendererProps<TableRowType>
   const { column, row, value, index } = props;
 
   const getNurseriesNames = (nurseries: any[]) => {
-    let nurseriesNames = '';
-    nurseries.forEach((n) => {
-      nurseriesNames += n.facility_name;
-    });
-    return nurseriesNames;
+    return (
+      <TextTruncated
+        stringList={nurseries.map((n) => n.facility_name)}
+        maxLengthPx={250}
```

Comment:
we could keep an eye on this and see if we need to provide some calculated width (some % of total width)?

Reply:
I'll move this up into a constant at least so that it's more visible